### PR TITLE
1946 handle even more cases of delayed document loading (scrollHeight…

### LINKED ISSUE #1946: Clicking on replies does not jump to the reply
For most of the replies in my reply tab, when I click on the reply, it just takes me to the top of the post and I have to scroll down to see the reply. I have reproduced on Firefox, Chrome, and Safari. It does not seem to happen with every reply though.

This example seems to happen every time I try it:
https://steemit.com/witness-category/@timcliff/the-reports-from-the-witnesses-2017-11-07#@justncase/re-timcliff-the-reports-from-the-witnesses-2017-11-07-20171107t184836970z

### PR DESCRIPTION
**Pull request by relativityboy**
Closes #1946 ...even more
… change) in a generic way

This update handles instances where the browser incorrectly sticks the viewport to the bottom of the page, or does not hold the scroll position at the correct element when images take longer to load.

### CODE REVIEW

File: src/app/utils/UserUtil.js [hunk 1]
```diff
@@ -21,11 +21,13 @@ export const setStore = (theStore) => {
  * @returns {boolean}
  */
 export const currentUser = () => {
-    const state = store.getState();
-    if(state.user) {
-        const current = state.user.getIn(['current']);
-        if(current) {
-            return current;
+    if (store) {
```
Comment (roadscape) [inline src/app/utils/UserUtil.js]: Shouldn't it be a hard requirement to call setStore before these methods? ie should it not be strict and throw an exception?

Comment (relativityboy) [reply src/app/utils/UserUtil.js]: That was my initial thought. 

@bnchdrff changed my mind about that. Maybe a console statement would be appropriate.

Comment (bnchdrff) [reply src/app/utils/UserUtil.js]: did you look into delaying scroll until content & images are done loading?

Comment (bnchdrff) [reply src/app/utils/UserUtil.js]: did you look into delaying scroll until content & images are done loading?

**Pull request merged**